app/controllers/Api/v1/posts_controller.rb

```diff
@@ -46,7 +46,7 @@ def reveal_post
 
   def secret
     if @post
-      render json: @post 
```

Comment:
ah .. is this required by slack ? This will break the current frontend. I will see how can modify it to suit this.

Reply:
No, I'm just returning the data which are only required